src/psychrometrics/p_sat_torr.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Estimates the saturation vapour pressure in [torr].
+ *
+ * Estimates the saturation vapour pressure in [torr].
```

Comment:
It looks like here there is a repetition in the code.

Reply:
Looks like duplication due to a messy merge - we'll remove that!